### PR TITLE
Update top-level README.md file to include a better description and a code snippet.

### PR DESCRIPTION
**Pull request by sushkoy**
## Description
Update top-level README.md file to include a better description and a code snippet. We'll have more documentation and 
examples on the web-site, but it's good to have some basic info in README.md

## How has this been tested?
A docs-only change, no tests.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+budget_accountant = pipeline_dp.NaiveBudgetAccountant(total_epsilon=1,
+                                                      total_delta=1e-6)
+
+# Wrap Spark's RDD into it's private version. You will use this private wrapper
+# for all further processing instead of the Spark's RDD. Using the wrapper ensures
+# that only private statistics can be released.
+private_movie_views = \
+    make_private(movie_views, budget_accountant, lambda mv: mv.user_id)
+
+# Calculate the private sum
```
Comment (dvadym) [inline README.md:38]: Nit: extend which exactly sum (sum of ratings per movie)

File: README.md [hunk 2]
```diff
@@ -1,9 +1,54 @@
...
+dp_result = private_movie_views.sum(
+    SumParams(max_partitions_contributed=2,
+              max_contributions_per_partition=2,
+              low=1,
+              high=5,
+              partition_extractor=lambda mv: mv.movie_id,
```
Comment (dvadym) [inline README.md:44]: Nit: add comment to this, somthing like specifies aggregation key

File: README.md [hunk 3]
```diff
@@ -1,9 +1,54 @@
...
+              value_extractor=lambda mv: mv.rating))
```
Comment (dvadym) [inline README.md:45]: Nit: add comment to this, something like, specifies what to sum

File: examples/movie_view_ratings_beam.py [hunk 4]
```diff
@@ -18,6 +18,10 @@
 
 def main(unused_argv):
     # Setup Beam
+
+    # Here, we use a local Beam runner.
+    # For a truly distributed calculation, connect to a Spark cluster (e.g.
```
Comment (dvadym) [inline examples/movie_view_ratings_beam.py:23]: Spark -> Beam (here is Beam code)

**Review by dvadym — COMMENTED** [review]
Thanks, it looks great, I left some comments!

File: README.md (hunk 1 above)
Comment (sushkoy) [reply README.md]: Good point, changed. Thinking about this more, a sum of ratings isn't a good metric. We should replace this example with "mean" once we have it.

File: README.md (hunk 2 above)
Comment (sushkoy) [reply README.md:45]: Done.

File: README.md (hunk 3 above)
Comment (sushkoy) [reply README.md]: Done.

**Review by dvadym — APPROVED** [review]

**Pull request merged**